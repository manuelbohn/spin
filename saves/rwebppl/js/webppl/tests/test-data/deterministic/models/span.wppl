[
  span(isEven, []),
  span(isEven, [2]),
  span(isEven, [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]),
  map(sum, span(isEven, mapN(idF, 1000))),
  map(sum, span(isEven, mapN(idF, 20000))),
  span(function(ar) { return isEven(ar.length) },
    [[],
     [1],
     [1, 2],
     [1, 2, 3],
     [1, 2, 3, 4],
     [1, 2, 3, 4, 5]
    ]
 )
]
