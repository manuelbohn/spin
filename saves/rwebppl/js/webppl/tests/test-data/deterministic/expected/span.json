{
  "result": [
    [[], []],
    [[2],[]],
    [[0,2,4,6,8],[1,3,5,7,9]],
    [249500, 250000],
    [99990000,100000000],
    [[[], [1,2], [1,2,3,4]], [ [1],[1,2,3],[1,2,3,4,5]]]
  ]
}
